FuelSDK/client.py

```diff
@@ -120,7 +123,28 @@ def __init__(self, get_server_wsdl = False, debug = False, params = None, tokenR
 
         self.wsdl_file_url = self.load_wsdl(wsdl_server_url, wsdl_file_local_location, get_server_wsdl)
 
-        ## get the JWT from the params if passed in...or go to the server to get it             
```

Comment:
we need to add validation for Auth Endpoint if the useOAuth2Authentication is set to true.

Reply:
Added validation.

Reply:
Added validation.